src/components/TrustConfirmation/index.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import settings from 'settings';
 import FormLayout from 'components/FormLayout';
-import TrustConfirmationForm from 'components/TrustConfirmationForm';
+import TrustSupportConfirmationForm from 'components/TrustSupportConfirmationForm';
+import TrustPublishConfirmationForm from 'components/TrustPublishConfirmationForm';
 
 const TrustConfirmation = ({
   isLoggedIn,
-  me
+  me,
+  action
 }) => (
   <FormLayout
     title={settings.trustConfirmationPage.title}
     intro={settings.trustConfirmationPage.intro}>
     <p>Thanks {me.firstname}, we sent an SMS verification code to <b>{me.mobile}</b></p>
```

Comment:
Ahh good spot. I've also got to do the button strings as well.
But i'll do this in the content / styling branch I guess.